cmd/openapi-gen/builder/openapi.go

```diff
@@ -99,17 +102,18 @@ func (o *openAPI) buildDescriptor(path string, consumes, produces []string, desc
 	}
 
 	for _, d := range descriptor.Children {
-		if err := o.buildDescriptor(descriptor.Path, c, p, &d); err != nil {
+		if err := o.buildDescriptor(specPaths, descriptor.Path, c, p, &d); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func (o *openAPI) buildPathItems(path string, consumes, produces []string, defs []definition.Definition) (map[string]spec.PathItem, error) {
-	specPaths := map[string]spec.PathItem{}
+func (o *openAPI) buildPathItem(consumes, produces []string, defs []definition.Definition) (*spec.PathItem, error) {
+	pathItem := spec.PathItem{}
+	hasDef := false
 	for _, def := range defs {
-		pathItem := spec.PathItem{}
+		hasDef = true
 		op, err := o.buildOperation(&def, consumes, produces, service.HTTPCodeFor(def.Method))
```

Comment:
Does it support **two or more** definitions with same method but have different consume-produce peer?

e.g.
```

definition.Definition {
    Method: Create,
    Consumes: []string{JSON},
    Produces: []string{JSON},
},
definition.Definition {
    Method: Create,
    Consumes: []string{File},
    Produces: []string{JSON},
},
```

Reply:
。。。

Reply:
what does `。。。`  mean?  not supported?